### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by bim9262**
Might want to consider vending a pre commit hook to help prevent so many typos in the future or a github check.

### CODE REVIEW

**Comment by ammgws** [comment]
Not sure the .vscode specific stuff should be in the repo.

Can you add some info to CONTRIBUTING.md as well?

**Comment by bim9262** [comment]
I'd be interested in hearing if there's another spell checker you'd prefer, before I add documentation on whatever tool seems like the best fit. I did take a look at https://crates.io/crates/cargo-spellcheck, but found the setup (hunspell) to be more difficult. cspell can be used in vscode (as an extension) or as a standalone utility from npm.

**Comment by cfsmp3** [comment]
Personally I think a PR that fixes existing typos would be good to merge, and table the rest for now.

I suspect many contributors don't use vscode at least for this specific project.

**Comment by bim9262** [comment]
Removed cspell config.

File: src/blocks/cpu.rs [hunk 1]
```diff
@@ -201,11 +201,11 @@ impl CpuTime {
         let idle = u64::from_str(s.next()?).ok()?;
         let iowait = u64::from_str(s.next()?).ok()?;
         let irq = u64::from_str(s.next()?).ok()?;
-        let softirq = u64::from_str(s.next()?).ok()?;
+        let soft_irq = u64::from_str(s.next()?).ok()?;
 
         Some(Self {
             idle: idle + iowait,
-            non_idle: user + nice + system + irq + softirq,
+            non_idle: user + nice + system + irq + soft_irq,
```
Comment (cfsmp3) [inline src/blocks/cpu.rs:208]: I'd leave softirq as written. See for example:

https://archive.kernel.org/oldlinux/htmldocs/kernel-hacking/basics-softirqs.html

Comment (bim9262) [reply src/blocks/cpu.rs]: Thanks the extra context!

**Pull request merged**